content/learn/contribute/project-information/bevy-organization.md

```diff
@@ -5,10 +5,10 @@ insert_anchor_links = "right"
 weight = 2
 +++
 
-The Bevy Organization is the group of people responsible for stewarding the Bevy Project. It handles things like merging pull requests, choosing project direction, managing bugs / issues / feature requests, running the Bevy website, controlling access to secrets, defining and enforcing best practices, etc.
+The Bevy Organization is the group of people responsible for stewarding the Bevy Project. It handles things like merging pull requests, choosing project direction, managing bugs / issues / feature requests, running Bevy's website, controlling access to secrets, defining, and enforcing best practices, etc.
 
 {% callout(type="info") %}
-Everyone is welcome and encouraged to contribute to Bevy, whether or not they're a Bevy Organization member. Community contributors (this means you) are encouraged to freely open issues, submit pull requests, and review pull requests.
+Everyone is welcome, and encouraged to contribute to Bevy, whether or not they're a Bevy Organization member. Community contributors (this means you) are encouraged to freely open issues, submit pull requests, and review pull requests.
```

Comment:
I don't think the current phrasing works. You can either remove the comma:

```suggestion
Everyone is welcome and encouraged to contribute to Bevy, whether or not they're a Bevy Organization member. Community contributors (this means you) are encouraged to freely open issues, submit pull requests, and review pull requests.
```

Or add another:

```suggestion
Everyone is welcome, and encouraged, to contribute to Bevy, whether or not they're a Bevy Organization member. Community contributors (this means you) are encouraged to freely open issues, submit pull requests, and review pull requests.
```

The current wording makes it seem like "and encouraged to contribute to Bevy" is an optional segment of the sentence, when in reality "contribute to Bevy" is really important.